### PR TITLE
feat(Loading) :  loading page

### LINKED ISSUE #8: Loading
Add loading when switching between pages, can be achieved by using `loading.tsx`

### PR DESCRIPTION
**Pull request by Vyshnav001**

### CODE REVIEW

File: src/app/loading.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default function Loading() {
+  return (
+    <div className="font-tiara text-6xl md:text-9xl min-h-screen flex items-center justify-center animate-pulse ">
+      <span className="text-tiara_red">T</span>iara{" "}
```
Comment (anishshobithps) [inline src/app/loading.tsx]: Please keep, `AR` as red instead of `T`

**Review by anishshobithps — CHANGES_REQUESTED** [review]

**Comment by anishshobithps** [comment]
Also does the Loading thing come up ?

**Comment by Vyshnav001** [comment]
> Also does the Loading thing come up ?


https://github.com/tiarasjec/website/assets/120462272/0c940da6-5d04-4987-a98c-d1612dde092d

**Review by anishshobithps — APPROVED** [review]
LGTM, wait for @rabilrbl 's review!

**Review by rabilrbl — APPROVED** [review]
LGTM

**Pull request merged**